test/test_dynamic_shapes.py

```diff
@@ -610,6 +610,68 @@ def test_method(self, fn, first_type, second_type):
 instantiate_parametrized_tests(TestSymNumberMagicMethods)
 
 class TestFloorDiv(TestCase):
+    @staticmethod
+    def python_floordiv(x, y):
+        return x // y
+
+    @staticmethod
+    def torch_floordiv(x, y):
+        # Note: we fully evaluate here since FloorDiv might not always do
+        # that.
+        shape_env = ShapeEnv()
+        return shape_env.evaluate_expr(FloorDiv(x, y))
```

Comment:
This assumes evaluate_expr actually evaluates fully, so this relies on earlier PRs in the stack